location/templates/location.html

```diff
@@ -40,6 +41,16 @@
 <h1>{{object.address}}, {{object.city}}, {{object.state}}, {{object.zipcode}}</h1>
 {% leaflet_map "leaflet_map_main" %}
 
+{% if request.user.is_authenticated %}
```

Comment:
This looks like its from the favorite location story. Are you making sure to keep your changes between branches separate?

Reply:
Reverted changes.